packages/headless/src/controllers/core/facets/category-facet/headless-core-category-facet.ts

```diff
@@ -170,10 +170,17 @@ export interface CoreCategoryFacetState {
 
   /**
    * The facet's values.
-   * @deprecated use `valuesAsTrees` instead.
+   * @deprecated use `selectedValueAncestry` instead.
    */
   values: CategoryFacetValue[];
 
+  /**
+   * The selected facet values ancestry.
+   * The first element is the "root" of the selected value ancestry tree.
+   * The last element is the selected value itself.
+   */
+  selectedValueAncestry: CategoryFacetValue[];
```

Comment:
Do we need the `selected` keyword here ? Since we don't allow exclusion, and the only way you can have ancestor here is by selecting a value in the first place.

Would `valueAncestry` be a correct replacement ?

Reply:
We don't need it.
However, technically, if you select a n-th level value, where n>1, you have multiple ancestor: indeed, each n-th leveled value other than 0 has ancestor.

So even if it's a bit redundant, I prefer to be abundantly clear than fuzzy, so IMHO, I'd keep the selected.